lib/types/index.d.ts

```diff
@@ -803,40 +803,135 @@ export type JSRuleDefinition<
 
 // #region Linter
 
+/**
+ * Class that is responsible for verifying JavaScript text.
```

Comment:
The linter is language-independent, so maybe "Class responsible for linting text or binary input".